index.js

```diff
@@ -70,6 +80,10 @@ function verify(model, options) {
     if (decoded == null) {
       // In this case something but not a valid JWT was provided to the
       // Auth Header, so we'll dismiss the request
+      if (options.useSailsResponses) {
+        return res.unauthorized();
```

Comment:
You didn't follow specs for responses:

    res.badRequest();

Or:

    return res.badRequest(data);

Or

    return res.badRequest(data, pathToView);